nerve/report.go

```diff
@@ -55,6 +55,9 @@ func (r *Report) UnmarshalJSON(b []byte) error {
 	if err := json.Unmarshal(b, &rr); err != nil {
 		return err
 	}
+	if rr.Port == 0 || rr.Host == "" || rr.Name == "" {
+		return errs.WithF(data.Fields{"report": rr}, "err: Malformed report")
```

Comment:
message is not explicit enough, and `err:` is useless

`Missing port, host or name in report definition` or similiar is better

Reply:
Fixed
I feel it's a bit redundant as we already have the info in report.

Reply:
You have the payload, but you don't know what the code do not like from it. Now you know.